pkg/apis/app/v2/kieapp_types.go

```diff
@@ -154,18 +154,39 @@ type KieServerSet struct {
 	// When set to true will configure the KIE Server image to disable all capabilities that are not related to decisions, Default to true for RHDM environments and false to RHPAM.
 	DecisionsOnly *bool `json:"decisionsOnly,omitempty"`
 	// Number of max KIE Executor sessions, it must be lower than the value of max-pool-size, by default is max-pool-size set to 60. Max pool size can be set by system property jboss.mdb.strict.max.pool.size (using javaOptsAppend: "-Djboss.mdb.strict.max.pool.size=40"), for more information see https://access.redhat.com/solutions/2955481.
-	KieExecutorMDBMaxSession *int `json:"kieExecutorMDBMaxSession,omitempty"`
+	KieExecutorMDBMaxSession *int              `json:"kieExecutorMDBMaxSession,omitempty"`
+	TerminationRoute         *TerminationRoute `json:"terminationRoute,omitempty"`
+}
+
+type TerminationRoute struct {
+	// Enable edge termination
+	EnableEdge bool `json:"enableEdge,omitempty"`
+	// Protocol to use, this could override the global disableSSL, http used with Edge enabled
+	// +kubebuilder:validation:Enum:=https;http
+	Protocol string `json:"protocol,omitempty"`
+	// +kubebuilder:validation:Enum:=passthrough;edge
+	// Passthrough is the default protocol, otherwise edge could be used.
+	Termination string `json:"termination,omitempty"`
+	// Enable the tls section when http port is used
+	EnableTlsSection bool `json:"enableTlS,omitempty"`
+	// Private Key if used on Edge termination (optional)
+	Key string `json:"key,omitempty"`
+	// Certificate if used on Edge termination (optional)
+	Certificate string `json:"certificate,omitempty"`
+	// CaCertificate if used on Edge termination (optional)
+	CaCertificate string `json:"caCertificate,omitempty"`
```

Comment:
What if the users are using OpenshiftCaBundle option? This is probably not supported, right? We should probably mentioned it in description

Reply:
Is not supported, in which section we could mention ?